sesman/sesman.ini

```diff
@@ -20,9 +20,10 @@ AlwaysGroupCheck=false
 ; Default: 10
 X11DisplayOffset=10
 
-;; MaxSessions - maximum number of connections to an xrdp server
+;; MaxSessions - maximum number of concurrent sessions
 ; Type: integer
 ; Default: 0
+; if positive value, limits the maximum number of concurrent sessions
```

Comment:
That repeats the line that starts with `;; MaxSessions`

I would rather say "If 0 or not set, the number of sessions is not limited"